client/lcd/test_helpers.go

```diff
@@ -114,7 +114,15 @@ func CreateAddr(t *testing.T, name, password string, kb crkeys.Keybase) (sdk.Acc
 // their respective sockets where nValidators is the total number of validators
 // and initAddrs are the accounts to initialize with some steak tokens. It
 // returns a cleanup function, a set of validator public keys, and a port.
-func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress) (func(), []crypto.PubKey, string) {
+func InitializeTestLCD(
+	t *testing.T, nValidators int, initAddrs []sdk.AccAddress,
+) (cleanup func(), valPubKeys []crypto.PubKey, valOperAddrs []sdk.ValAddress, port string) {
+
+	// TODO: Allow caller to specify respective power for each validator
```

Comment:
I'm not sure we want this TODO - seems like it would make tests confusing to write... where would we need this?

Reply:
I guess testing specific setups. But I wont do this for now and will remove it and just address what you state below.